DPNUnet.imjoy.html

```diff
@@ -513,7 +516,81 @@
             await api.showProgress(100)
             api.alert(f'Prediction done, generated masks saved to {config.dataset_path}')
 
+    async def predict(self,ctx):
+        '''
+        API function to perform nuclei segmentation on user provided data. 
+        Data has to be stored on the plugin engine where DPNUnet plugin is running.
+        Segmented nuclei will also be converted in GeoJson regions, and (if requested)
+        a download link with the zipped results be provided. 
+
+        Data has to be stored in the following format:
+        - Each image has to be in its own folder (sample folder)
+        - Sample folders must be stored in parental folder called 'test'
+        - Each image has to have the same name (e.g. 'nuclei.png')
+
+        Args:
+            ctx.data.data_path (string): folder containing data to be segmented (folder containing subfolder 'test').
```

Comment:
The variable `ctx` is more for `run` function, here as a standard plugin api function, maybe a better idea is to replace `ctx.data` to `config` in your case. so you will essentially pass `config={'data_path': .., 'image_file_name': ..., 'zip_name': ...}`

Reply:
ok. will change that in next commit.